### DISCUSSION TITLE
over 3000 plots and never getting rewards

### DISCUSSION THREAD

**Discussion by GCNZ18** [Chia-Network/chia-blockchain#12597] [Support]
any advice on why i never get a reward with over 3000 plots
I have run the plots checker and doesnt seem to be anything wrong with the plots

**Comment by loppefaaret** [comment]
how long a timeframe are we talking here ? with 300TiB~ and really bad luck, it could be ½ a year

**Reply by GCNZ18** [reply]
over 6 months
Farming status: Farming
Total chia farmed: 0.5
User transaction fees: 0.0
Block rewards: 0.5
Last height farmed: 2247068
Local Harvester
   3968 plots of size: 415.409 TiB
Plot count for all harvesters: 3968
Total size of plots: 415.409 TiB
Estimated network space: 21.811 EiB
Expected time to win: 1 week and 5 days
